app/admin/simulator/slack/page.tsx

```diff
@@ -1,17 +1,269 @@
-import SimulatorLayout from "@/components/SimulatorLayout";
+"use client";
+
+import React, { useState } from "react";
+import { HashtagIcon, LockClosedIcon, PlusIcon, ChevronDownIcon, ChevronRightIcon } from "@heroicons/react/24/outline";
+
+interface Channel {
+  id: string;
+  name: string;
+  isPrivate: boolean;
+  unreadCount?: number;
+}
+
+interface Message {
+  id: string;
+  user: string;
+  avatar: string;
+  timestamp: string;
+  content: string;
+}
+
+const mockChannels: Channel[] = [
+  { id: "1", name: "general", isPrivate: false, unreadCount: 3 },
+  { id: "2", name: "random", isPrivate: false },
+  { id: "3", name: "development", isPrivate: false, unreadCount: 1 },
+  { id: "4", name: "design", isPrivate: false },
+  { id: "5", name: "sales-alerts", isPrivate: true, unreadCount: 2 },
+  { id: "6", name: "admin-only", isPrivate: true },
+];
+
+const mockMessages: Message[] = [
+  {
+    id: "1",
+    user: "David Vargas",
+    avatar: "DV",
+    timestamp: "10:30 AM",
+    content: "Hey team! Just wanted to update everyone on the VargasJR development progress. We've made some great improvements to the agent system."
+  },
+  {
+    id: "2",
+    user: "Sarah Chen",
+    avatar: "SC",
+    timestamp: "10:32 AM",
+    content: "That's awesome! Can you share more details about the improvements?"
+  },
+  {
+    id: "3",
+    user: "Mike Johnson",
+    avatar: "MJ",
+    timestamp: "10:35 AM",
+    content: "I'm particularly interested in the performance optimizations 🚀"
+  },
+  {
+    id: "4",
+    user: "David Vargas",
+    avatar: "DV",
+    timestamp: "10:37 AM",
+    content: "Sure! We've improved the agent deployment speed by 40% and added better error handling. The new UI simulator is also looking great."
+  },
+  {
+    id: "5",
+    user: "Emily Rodriguez",
+    avatar: "ER",
+    timestamp: "10:40 AM",
+    content: "Love the progress! When can we expect the next release?"
+  }
+];
 
 export default function SlackSimulatorPage() {
+  const [selectedChannel, setSelectedChannel] = useState<Channel>(mockChannels[0]);
+  const [sidebarOpen, setSidebarOpen] = useState(false);
+  const [channelsExpanded, setChannelsExpanded] = useState(true);
+  const [directMessagesExpanded, setDirectMessagesExpanded] = useState(true);
```

Comment:
Move this state (and all related components and data) into a new client page that gets imported here